tests/controllers/qos_controller_test.cpp

```diff
@@ -37,8 +41,13 @@ TEST(SerenityControllerTest, NoQoSCorrections) {
   Try<Nothing> initialize = controller->initialize(
       lambda::bind(&MockSlaveUsage::usage, &usage));
 
+  process::Clock::pause();
```

Comment:
Sure (:

The QoSController spawns new actor (process) and run 'corrections' within it. This process uses (defers a callback) on *usage()* function defined in this test body. Since this test expects __NO__ corrections, without clock pause and settle, it will end immediately, before the QoSController actor even tries to run *usage()*. We are settling clock to empty libprocess queue and give a chance for the QoSController actor to use *usage()* and finish its work. Otherwise we are getting segmentation fault, since the QoSController is trying to use *usage()* which is now undefined because gtest is doing next tests in main thread... ((: